### DISCUSSION TITLE
Mantis "list" command

### DISCUSSION THREAD

**Discussion by 0xbharath** [PhonePe/mantis#29] [Ideas]
@Prateek-Thakare @Ncoder2  Can we have a "list" as a top level command for mantis along with onboard and scan? I feel that this can come in handy when one wants to list orgs that are onboarded to the Mantis and other details. I have onboarded 20+ orgs on Mantis and it is not comfortable to do a DB query or go to dashboard everytime to get this list.

Example:

```
$ mantis list orgs
org1 
org2
org2
```

**Comment by Prateek-Thakare** [comment]
Hey @0xbharath 
This feature was added as part of latest commit.

**Reply by 0xbharath** [reply]
Thanks. Will check it out.

**Reply by 0xbharath** [reply]
This works as expected. Thank you. However, there is an unhandled exception when no orgs are onboarded. Can you add exception handling for these cases?
